### DISCUSSION TITLE
Review did not load

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#46272] [Mobile] [answered]
GitHub Mobile beta

1. Open notifications
    ![Screenshot_20230203-082901.png](https://user-images.githubusercontent.com/2119212/216615437-bf1f7676-5738-4fbf-8983-e6f494bfd123.png)
2. Tap the scalaz one https://github.com/scalaz/scalaz/pull/2381
3. See an empty view 
    ![Screenshot_20230203-082726.png](https://user-images.githubusercontent.com/2119212/216615583-71a636b9-0c50-4c50-ab69-19ded3f46658.png)
3. Use a back button and browse to it again and things rendered properly. 

Expected results:
* Don't show empty views.
* Include some loading placeholder (a spinner?) and handling for when loading fails

**Comment by alcere** [comment, marked as answer]
Hi @jsoref. I could reproduce the issue. The team will work on a fix as soon as possible. Please, stay tuned for updates!
